File: observer/Samuel Sales/DesafioObserver_Samuel/src/Revistas/Veja.java
package Revistas;

import agencia.Agencia;

/**
 *
 * @author Samuel Sales - Facul
 */
public class Veja implements ObserverEditoraAbril {

    @Override
    public void atualizar(Agencia entrada) {
        System.out.println(entrada.getNome() + ": Revista Veja");
    }

}

File: observer/PadraoObserverII/PadraoObserverII/Revistas/Veja.cs
using PadraoObserverII.Interface;
using System;

namespace PadraoObserverII.Revistas
{
    class Veja : IObserverRevista
    {
        public void Atualizar(Agencia entrada)
        {
            Console.WriteLine(entrada.Nome + ": Notícias da Revista Veja!");
        }
    }
}
